samples/bluetooth/central_uart/README.rst

```diff
@@ -15,6 +15,37 @@ On Nordic Semiconductor's development kits, the UART 1 peripheral is typically g
 
 Any data sent from the Bluetooth LE unit is sent out of the UART 1 peripheral's TX pin.
 
+Debugging
+*********
+
+Debug messages are printed by RTT logger.
+In this sample UART console is used to send/read data over the NUS Client.
+You will be able to display RTT console messages by doing the following:
+
+Windows
+=======
+
+* Open the "J-Link RTT Viewer" application
+* Select the following options:
+
+  * Connection: USB
+  * Target Device: Select your IC from the list
+  * Target Interface and Speed: SWD, 4000 KHz
+  * RTT Control Block: Auto Detection
+
+GNU/Linux and macOS (OS X)
+==========================
+
+* Open ``JLinkRTTLogger`` from a terminal
+* Select the following options:
+
+  * Device Name: Use the fully qualified device name for your IC
+  * Target Interface: SWD
+  * Interface Speed: 4000 KHz
+  * RTT Control Block address: auto-detection
+  * RTT Channel name or index: 0
+  * Output file: filename or ``/dev/stdout`` to display on the terminal directly
```

Comment:
You can refer to _rtt_connect section that is being added in this PR #2502

Reply:
Yes, sure. Done

Reply:
now, it's _testing_rtt_connect